### PR TITLE
color filter : double-click to reset

### LINKED ISSUE #11854: double click on new color labels filters do not reset
When double clicking on the color labels one expect the filter to be reset as it is the case for stars. But somehow this got broken recently and does not work on my side.

### PR DESCRIPTION
**Pull request by AlicVB**
this fix #11854

### CODE REVIEW

File: src/libs/filters/colors.c [hunk 1]
```diff
@@ -54,6 +54,14 @@ static void _set_mask(dt_lib_filtering_rule_t *rule , const int mask, const gboo
 
 static gboolean _colors_clicked(GtkWidget *w, GdkEventButton *e, _widgets_colors_t *colors)
 {
+  // double click reset the widget
+  if(e->button == 1 && e->type == GDK_2BUTTON_PRESS)
+  {
+    _set_mask(colors->rule, 0, TRUE);
```
Comment (phweyland) [inline src/libs/filters/colors.c:60]: _colors_update() does it anyway, but clearer to set the default as CL_AND_MASK (and).

**Review by phweyland — COMMENTED** [review]
Otherwise fine for me.

File: src/libs/filters/colors.c (hunk 1 above)
Comment (AlicVB) [reply src/libs/filters/colors.c]: done

**Review by TurboGit — APPROVED** [review]
Works for me, thanks!

**Pull request merged**